eng/common/scripts/git-branch-push.ps1

```diff
@@ -27,7 +27,8 @@ param(
     [Parameter(Mandatory = $false)]
     [string] $PushArgs = "",
 
-    [string] $RemoteName = "azure-sdk-fork",
+    [Parameter(Mandatory = $false)]
+    [string] $RemoteName = "",
```

Comment:
You should be keeping the default remote name I don't understand a scenario where we ever need pass a different name. We might even be able to completely remove this parameter because I don't believe we should ever need it except for is some rare care were we want to use an exact remote we already set. I don't believe we should ever really use origin here. As we saw unless the remote was set up the exact matching giturl this script will fail so we might as well always create a new remote that exactly matches the giturl.